lua/wikis/chess/MatchSummary.lua

```diff
@@ -0,0 +1,197 @@
+	local showCountdown = match.timestamp ~= DateExt.defaultTimestamp
+
+	return MatchSummaryWidgets.Body{children = WidgetUtil.collect(
+		showCountdown and MatchSummaryWidgets.Row{children = DisplayHelper.MatchCountdownBlock(match)} or nil,
+		--need the match available in map display instead of just the date so we can access the links
+		Array.map(match.games, createGame),
+		MatchSummaryWidgets.Casters{casters = match.extradata.casters},
+		CustomMatchSummary._linksTable(match)
+	)}
+end
+
+---@param game MatchGroupUtilGame
+---@param gameIndex integer
+---@return MatchSummaryRow
+function CustomMatchSummary.createGame(game, gameIndex)
+	return MatchSummaryWidgets.Row{
+		classes = {'brkts-popup-body-game'},
+		css = {padding = '4px'},
+		children = WidgetUtil.collect(
+			-- Header
+			CustomMatchSummary._getHeader(game),
+
+			-- Player 1
+			CustomMatchSummary._getSideIcon(game.opponents[1]),
+			MatchSummaryWidgets.GameWinLossIndicator{winner = game.winner, opponentIndex = 1},
+			MatchSummaryWidgets.GameTeamWrapper{flipped = false},
+
+			-- Center
+			MatchSummaryWidgets.GameCenter{
+				css = {
+					['text-align'] = 'center',
+					['align-content'] = 'center',
+					['min-height'] = '1.5rem',
+					['font-size'] = '85%',
+					['line-height'] = '0.75rem',
+					['max-width'] = '200px'
+				},
+				children = CustomMatchSummary._getCenterContent(game, gameIndex),
+			},
+
+			-- Player 2
+			MatchSummaryWidgets.GameTeamWrapper{flipped = true},
+			MatchSummaryWidgets.GameWinLossIndicator{winner = game.winner, opponentIndex = 2},
+			CustomMatchSummary._getSideIcon(game.opponents[2]),
+
+			-- Comment
+			MatchSummaryWidgets.GameComment{classes = {'brkts-popup-sc-game-comment'}, children = game.comment}
+		)
+	}
+end
+
+---@param game MatchGroupUtilGame
+---@param gameIndex integer
+---@return Widget
+function CustomMatchSummary._getCenterContent(game, gameIndex)
+	return Div{
+		children = {
+			Span{
+				classes = {'brkts-popup-spaced'},
+				children = {
+					'Game ' .. gameIndex,
+					tonumber(game.length) and (' - ' .. game.length .. ' moves') or '',
+				},
+			},
+			Span{
+				classes = {'brkts-popup-spaced'},
+				css = {['font-size'] = '85%'},
+				children = {Eco.getName(game.extradata.eco, true)},
+			},
+		},
+	}
+end
+
+---@param gameOpponent table
+---@return Widget
+function CustomMatchSummary._getSideIcon(gameOpponent)
+	return Div{
+		classes = {'brkts-popup-spaced'},
+		css = {['padding'] = '0px 4px'},
+		children = KING_ICONS[gameOpponent.color],
+	}
+end
+
+---@param game MatchGroupUtilGame
+---@return Widget?
+function CustomMatchSummary._getHeader(game)
+	return String.isNotEmpty(game.header) and {
+		Div{
+			children = game.header,
+			css = {
+				['font-weight'] = 'bold',
+				['font-size'] = '85%',
+				margin = 'auto'
+			}
+		},
+		MatchSummaryWidgets.Break{}
+	} or nil
+end
+
+---@param match any
+---@return Widget?
+function CustomMatchSummary._linksTable(match)
```

Comment:
If there's no links, the hide/show row will still be shown. It should probably be hidden if there's no links